Packages/Core/Editor/Editors/AtomEventEditor.cs

```diff
@@ -12,6 +16,9 @@ namespace UnityAtoms.Editor
     public abstract class AtomEventEditor<T, E> : UnityEditor.Editor
         where E : AtomEvent<T>
     {
+#if UNITY_2018_4_OR_NEWER
```

Comment:
Same as above. The UNITY_2019_1_OR_NEWER condition should be before UNITY_2018_4_OR_NEWER, the second if-statement will never be used when running Unity 2019 since the UNITY_2018_4_OR_NEWER flag will also be true in this case.